app/helpers/where_helper.rb

```diff
@@ -0,0 +1,10 @@
+module WhereHelper
+  def where_or_stub_for(user)
+    user.last_whereabouts ||
+      OpenStruct.new(body: I18n.t('shruggie'), provenance: nil, sent_at: nil)
+  end
+
+  def reply_subject(where)
+    "Re: #{where.body.split("\n").first || where.body}"
```

Comment:
The only case when `where.body.split("\n").first` is falsey is when `where.body` is empty, so you can get rid of the alternative here.